digitalearthau/config/eo3/products-aws/ga_s2_wo_3.odc-product.yaml

```diff
@@ -0,0 +1,85 @@
+---
+name: ga_s2_wo_3
+metadata_type: eo3
+description: Sentinel 2 Water Observations
```

Comment:
Please add a license, @MatthewJA. And a bit of whitespace. See the WO product def :-)